sass/components/_redButton.scss

```diff
@@ -0,0 +1,15 @@
+.redButton {
+  width: 35rem;
+  height: 8rem;
+  border-radius: 16px;
+  background-color: #ff0000;
+  color: #fff;
+  font-weight: 500;
+  font-size: 2.6rem;
+  text-transform: uppercase;
+  outline: none;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 0px 4px 30px #ff0000;
+  align-self: flex-end;
+  border: none;
+  font-family: Montserrat;
+}
```

Comment:
Brakuje ":hover"

Reply:
Moje zadanie obejmuje :
- Komponent zdefiniowany w odrębnym pliku .js
- Komponent przyjmuje na wejściu napis do wyświetlenia.
- Komponent wyświetla przekazany napis, wyśrodkowany i dużymi literami.
- Działanie komponentu zostało pokryte testami jednostkowymi.

oraz

- **Wygląd komponentu wg. projektu,** który nie obejmuje hover

Dlatego hover nie dodałam, ale jeżeli Twoim zdaniem powinnam to oczywiście od razu to zrobię

Reply:
Zmiany dodane

Reply:
O tym nie pomyślałem, że nie było wspomniane - przepraszam! ;)